daemon/algod/api/algod.oas2.json

```diff
@@ -1264,6 +1264,12 @@
               "$ref": "#/definitions/SimulateRequest"
             }
           },
+          {
+            "description": "The boolean flag that lifts the limit on log opcode during simulation.",
+            "name": "lift-log-limits",
+            "in": "query",
```

Comment:
You may already be working on this, but I think it makes more sense for this to be in `SimulateRequest` now

Reply:
oh right, good catch

Reply:
see [move lift-log-limits to SimulationRequest](https://github.com/algorand/go-algorand/pull/5247/commits/ec83ba9d01ed3e5f31ad48d6349d4d589b5c0995)